receiver/apachereceiver/metadata.yaml

```diff
@@ -78,12 +78,12 @@ metrics:
     attributes: [ server_name ]
   apache.scoreboard:
     enabled: true
-    description: The number of connections in each state.
+    description: The number of workers in each state.
```

Comment:
I don't believe this change is accurate. My understanding is that the scoreboard represents workers, some of which may have active connections, but not all.

Reply:
According to https://metacpan.org/pod/Apache::Scoreboard, it seems it represents server or workers:

> Apache keeps track of server activity in a structure known as the scoreboard . There is a slot in the scoreboard for each child server and its workers (be it threads or processes), containing information such as status, access count, bytes served and cpu time, and much more.